From 49ccd9dbacdf914dd7662f5f3bacc8796e155f2f Mon Sep 17 00:00:00 2001
From: David Pacheco <dap@joyent.com>
Date: Mon, 9 Oct 2017 11:37:41 -0700
Subject: [PATCH] MANTA-3459 manta-adm manual page could use example of listing
 compute ids and storage ids Reviewed by: Joshua M. Clulow <jmc@joyent.com>
 Reviewed by: Jordan Hendricks <jordan.hendricks@joyent.com> Approved by:
 Joshua M. Clulow <jmc@joyent.com>

---
 docs/man/man1/manta-adm.md | 4 ++++
 man/man1/manta-adm.1       | 9 +++++++++
 2 files changed, 13 insertions(+)

diff --git a/docs/man/man1/manta-adm.md b/docs/man/man1/manta-adm.md
index 4231665..b1b9927 100644
--- a/docs/man/man1/manta-adm.md
+++ b/docs/man/man1/manta-adm.md
@@ -471,6 +471,10 @@ Example: list hostnames in form suitable for "sdc-oneachnode -n":
 
     # manta-adm cn -n
 
+Example: list storage CNs with their associated storage id (used in object
+metadata) and compute ids (used by the compute jobs subsystem):
+
+    # manta-adm cn -o host,admin_ip,compute_id,storage_ids storage
 
 ### "genconfig" subcommand
 
diff --git a/man/man1/manta-adm.1 b/man/man1/manta-adm.1
index acd8e0d..768304d 100644
--- a/man/man1/manta-adm.1
+++ b/man/man1/manta-adm.1
@@ -506,6 +506,15 @@ Example: list hostnames in form suitable for "sdc\-oneachnode \-n":
 # manta\-adm cn \-n
 .fi
 .RE
+.PP
+Example: list storage CNs with their associated storage id (used in object
+metadata) and compute ids (used by the compute jobs subsystem):
+.PP
+.RS
+.nf
+# manta\-adm cn \-o host,admin_ip,compute_id,storage_ids storage
+.fi
+.RE
 .SS "genconfig" subcommand
 .PP
 \fB\fCmanta\-adm genconfig "lab" | "coal"\fR
-- 
2.21.0

